src/meshpy/abaqus/input_file.py

```diff
@@ -22,15 +22,18 @@
 """This module defines the class that is used to create an input file for
 Abaqus."""
 
-from enum import Enum, auto
+from enum import Enum as _Enum
+from enum import auto as _auto
```

Comment:
Do want to leave it like this?

```suggestion
from enum import Enum as _Enum, auto as _auto
```

this would be more concise

Reply:
I would also prefer your solution (that's actually how I changed it), but the formatter apparently does not like it.

Reply:
You're right - somehow ruff pulls apart everything...

I looked into it if there is a magical setting - but I was not able to find anything. Also chatgpt did not "know" it

So if this is the standard - let's keep it this way so the magical linter and formatter is happy :D